File: Go/002-even-fibonacci-numbers.go
package main

import "fmt"

func main() {
	i, c, sum := [...]int{1, 1}, 0, 0
	for c < 4e6 {
		c = i[0] + i[1]
		i[0] = i[1]
		i[1] = c
		if c%2 == 0 {
			sum += c
		}
	}
	fmt.Printf("Fibonacci of terms is equal to %d\n", sum)
}

File: C/002-even-fibonacci-numbers.c
#include <stdio.h>
#include <stdlib.h>

long long fibonacci(int n);

int main(void)
{
    int i;
    int n=50;
    for (i=0; i<=n; i++)
        printf("Fibonacci of %02i is equal to %lld\n", i, fibonacci(i));

    return 0;
}

long long fibonacci(int n) {
    if (n == 0)
        return 0;
    else if (n == 1)
        return 1;
    else
        return (fibonacci(n-1) + fibonacci(n-2));
}

